package.json

```diff
@@ -176,6 +176,30 @@
                             "description": "%configuration.java.test.config.workingDirectory.description%",
                             "default": "${workspaceFolder}"
                         },
+                        "classPaths": {
```

Comment:
should add `modulePaths` as well.

Reply:
Yes, there will be another PR to solve the module path  issue: #1101

Reply:
i assume you just pass these configs to debugger directly. Is there anything to block you adding modulePaths at the moment?

Reply:
No, just want to have two PRs because they are two different issues. But I'm also fine to address them together in this single PR.